index.html

```diff
@@ -926,26 +920,26 @@
     </p>
     <ul>
       <li>
-        Only <a>browsing context</a>s with a defined <a>origin</a>, whose schema
-        is starting with <code>"https"</code> must be able to access
+        Only <code>
+        <a href="https://html.spec.whatwg.org/#document">Document</a></code>s
```

Comment:
It's not really the Document that's accessing anything. It's script in an execution environment (https://html.spec.whatwg.org/#script-execution-environment).